js/src/3rdparty/email-verification/index.js

```diff
@@ -29,6 +29,20 @@ export const isServerRunning = (isTestnet = false) => {
     });
 };
 
+export const hasReceivedCode = (email, address, isTestnet = false) => {
+  const port = isTestnet ? 28443 : 18443;
+  const query = stringify({ email, address });
+  return fetch(`https://email-verification.parity.io:${port}/?` + query, {
```

Comment:
1. Blank line before
2. Why are we using template strings & ES6-style string +'s in one?